### PR TITLE
release 1.29.0

### PR DESCRIPTION
**Pull request by MIchaelMainer**

### CODE REVIEW

File: src/Core/GraphConstants.php [hunk 1]
```diff
@@ -23,7 +23,7 @@ final class GraphConstants
     const REST_ENDPOINT = "https://graph.microsoft.com/";
 
     // Define HTTP request constants
-    const SDK_VERSION = "1.28.0";
+    const SDK_VERSION = "1.29.0";
```
Comment (Ndiritu) [inline src/Core/GraphConstants.php:26]: I believe this should still be 1.28.0? Although I see we have a 1.29.0 on Packagist and I'm not sure how that got there.

**Review by zengin — APPROVED** [review]

**Review by Ndiritu — CHANGES_REQUESTED** [review]
One comment on SDK version.
Also looks like we need to update `dev` with the latest changes from `main` in order to merge?

File: src/Core/GraphConstants.php (hunk 1 above)
Comment (MIchaelMainer) [reply src/Core/GraphConstants.php:26]: Yep, I made a mistake and didn't realize that I had already bumped the number but hadn't released. I tagged it before pushing to main which resulted in automatic publish to packagist.

Comment (Ndiritu) [reply src/Core/GraphConstants.php:26]: Cool, makes sense

**Pull request merged**